backend/src/main/kotlin/app/ehrenamtskarte/backend/common/webservice/WebService.kt

```diff
@@ -3,18 +3,23 @@ package app.ehrenamtskarte.backend.common.webservice
 import io.javalin.Javalin
 import io.javalin.http.staticfiles.Location
 
-const val PORT = 7000
+const val DEFAULT_PORT = "7000"
 
 class WebService {
     fun start() {
+        val production = System.getProperty("app.production", "").isNotEmpty()
+        val host = System.getProperty("app.host", "0.0.0.0")
+        val port = Integer.parseInt(System.getProperty("app.port", DEFAULT_PORT))
         val app = Javalin.create { cfg ->
-            cfg.enableDevLogging()
-            cfg.enableCorsForAllOrigins()
-            cfg.addStaticFiles("/graphiql", "/graphiql", Location.CLASSPATH)
-        }.start(PORT)
+            if (!production) {
+                cfg.enableDevLogging()
+                cfg.enableCorsForAllOrigins()
+                cfg.addStaticFiles("/graphiql", "/graphiql", Location.CLASSPATH)
+            }
```

Comment:
do we disable graphiql in production? I'd probably vote for having it enabled :D

Reply:
I think this is a tool for debugging and should not be exposed in production. Also you can always use intellij for sending graphql queries.

Not sure whether this tool is intended for production and is tested against vulnerabilities.

Reply:
To me, its just a public access to the api, which shouldn't do any harm...

Reply:
I see this more like a standalone application which could be deployed to production. I mean it is like the administration backend. I added it again because there is no problem from security perspective as these are just static files.